src/sentry/mail/forms/member_team.py

```diff
@@ -63,17 +63,22 @@ def clean(self) -> dict[str, Any] | None:
             self.add_error("targetIdentifier", msg)
             return
 
-        if targetType == self.memberValue and not user_service.get_many(
-            filter={
-                "is_active": True,
-                "is_active_memberteam": True,
-                "organization_id": self.project.organization.id,
-                "project_ids": [self.project.id],
-                "user_ids": [int(targetIdentifier)],
-            }
-        ):
-            msg = forms.ValidationError("This user is not part of the project.")
-            self.add_error("targetIdentifier", msg)
-            return
+        if targetType == self.memberValue:
+            is_active_team_member = OrganizationMemberTeam.objects.filter(
+                is_active=True,
```

Comment:
This looks like it dropped the `is_active_memberteam` condition? From UserFilterQuery:
```
if "is_active_memberteam" in filters:
    query = query.filter(
        sentry_orgmember_set__organizationmemberteam__is_active=filters[
            "is_active_memberteam"
        ],
    )
```

Reply:
Correct, this query from control silo is no longer possible due to breaking the FK.  The result is this logic, where we need to query the OrganizationMemberTeam from region and combine that with the user queries.